.github/workflows/benchmarks.yml

```diff
@@ -36,24 +36,24 @@ jobs:
           node-version: "lts/*"
 
       - name: Install Ruby and related tools
-        run: ./scripts/benchmarks/setup_dependencies.sh
+        run: ./scripts/benchmarks/setup_ruby.sh
 
       - name: Install cargo tools
         run: |
-          cargo install nj-cli wasm-pack --locked
+          cargo install nj-cli wasm-pack
           npm install -g tslib
 
       - name: Run Jasmine performance tests
         run: |
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-          export PATH="/root/.cargo/bin:$PATH"
-          source ~/.bashrc
-          if [[ "${{ github.event_name }}" == 'workflow_dispatch' && "${{ github.event.inputs.pr_id }}" -eq 0 ]]; then
-            echo "Running manually"
-            ruby scripts/benchmarks/process.rb 1
+          rustup -V
```

Comment:
This is the command to get the version of rust toolchain manager and not rust version itself. Although they are printing rust version now but there is no guarantee this will still the case. To get Rust version we can use `rustc --version`